pom.xml

```diff
@@ -275,37 +275,7 @@
                 <artifactId>maven-jar-plugin</artifactId>
                 <version>3.1.1</version>
             </plugin>
-            <plugin>
```

Comment:
Why did you remove this? (Especially the javadoc one)

Reply:
I readded the javadoc plugin.

Reply:
Oops. I take those out to speed up compilation locally. They should go back in!